cli/tauri.js/test/jest/fixtures/app-test-setup.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path')
+const process = require('process')
+
+const appDir = path.resolve(__dirname, '../fixtures/app')
+const tauriDir = path.join(appDir, 'src-tauri')
+
+module.exports.appDir = appDir
+module.exports.distDir = path.join(appDir, 'dist')
+
+import * as appPaths from '../../../src/helpers/app-paths'
+
+module.exports.initJest = () => {
+  jest.mock('helpers/non-webpack-require', () => {
```

Comment:
You should add a comment in helpers/non-webpack-require explaining that it is a helper so that it can be mocked.